docs/federate.md

```diff
@@ -43,6 +43,22 @@ and sends them back to the user as a single answer.
 In your command line, create and navigate to a new directory where you will keep the configuration files for your new `f-API`. 
 In this directory, create three files:
 
+!!! info "Or, clone and use our configuration template repository"
+    The [neurobagel/recipes](https://github.com/neurobagel/recipes) repository contains templates of all files needed for configuring different types of Neurobagel deployments.
+    To get the three files you need to set up local federation (described in detail in the sections below), run the following code:
+
+    ```bash
+    git clone https://github.com/neurobagel/recipes.git
+    cd local_federation  # for local federation, ensure you only edit files inside this subdirectory
+
+    # make a non-template copy of the files you will need to edit
+    cp local_nb_nodes.template.json local_nb_nodes.json
+    cp template.env .env
+    ```
+
+    Then, edit `local_nb_nodes.json` and `.env` in your `local_federation/` directory as needed for your specific deployment, based on the instructions in the below sections.
```

Comment:
how about we just make this the default? I feel like giving our readers too many choices here when they are likely already a bit confused is not going to help them much.